exchange/auction.go

```diff
@@ -15,6 +16,29 @@ import (
 	"github.com/prebid/prebid-server/prebid_cache_client"
 )
 
+type DebugLog struct {
+	EnableDebug bool
+	CacheType   prebid_cache_client.PayloadType
+	Data        DebugData
+	TTL         int64
+	CacheKey    string
+	CacheString string
+}
+
+type DebugData struct {
+	Request  string
+	Headers  string
+	Response string
+}
+
+func (d *DebugLog) BuildDebugLog() {
```

Comment:
Consider renaming to something like `BuildCacheString` to make it clear what field is being modified.

Reply:
Renamed to `BuildCacheString`